src/core/annotation.js

```diff
@@ -1966,16 +1974,21 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
   getFieldObject() {
     let type = "button";
     let value = null;
+    let exportValues;
     if (this.data.checkBox) {
       type = "checkbox";
       value = this.data.fieldValue && this.data.fieldValue !== "Off";
+      exportValues = this.data.exportValue;
```

Comment:
I believe this is incorrect, since according to https://www.adobe.com/content/dam/acom/en/devnet/acrobat/pdfs/js_api_reference.pdf#page=429&zoom=auto,-207,718 the export values for the field object must always be an array of strings. I recomment to change `let exportValues;` above to `let exportValues = []` since it doesn't seem to be optional and in these branches simply push the applicable values to the array (they are guaranteed to be strings). That way this field is also valid (empty) for e.g., push buttons.

Moreover, from the spec descrption I _think_ for checkboxes the list needs to include _all_ export values. `this.data.exportValue` is only the "not Off" value, so shouldn't this be `exportValues = ["Off", this.data.exportValue]`? Please check what e.g. Adobe Reader expects here.

Finally, please move the export value logic to a second commit since it's separate from adding default values.